publish/publish.proj

```diff
@@ -185,6 +187,23 @@
     </ItemGroup>
     <Error Condition="'@(_DownloadedSymbolsPackages)' != '' and '$(NuGetSymbolsFeedUrl)' == ''" Text="Missing required property NuGetSymbolsFeedUrl" />
     
+  </Target>
+
+  <Target Name="GetPackagesToSign">
+    <ItemGroup>
+      <FilesToSign Include="@(_DownloadedStandardPackages)">
+        <Authenticode>NuGet</Authenticode>
+      </FilesToSign>
+    </ItemGroup>
+    <Message Importance="High" Text="Attempting to sign package '%(FilesToSign.Identity)'" />
+  </Target>
+
+  <Target Name="SignPackages"
+          Condition="'$(SkipSigning)' != 'true' and '$(SignType)' != 'public'"
+          DependsOnTargets="GetPackagesToSign;SignFiles">
+  </Target>
+
+  <Target Name="DoPushCoreHostPackagesToFeed">
```

Comment:
(nit) Can we drop the `Do` in the name?  `PushCoreHostPackagesToFeed` seems good enough to me.

Reply:
`Do` also makes me read the rest as if it's a question. 😄

I think adding in `Signed` would make the sequence look good:

`PublishCoreHostPackagesToFeed` depends on:
`GetPackagesToSign;SignPackages;PushSignedCoreHostPackagesToFeed`.